site/aktuelles/index.html

```diff
@@ -1,14 +1,14 @@
 <!-- HERO -->
 <div class="row">
-    <div class="hero hero--image hero--light"
+    <div class="hero hero--image hero--small hero--light"
         style="background: linear-gradient(0deg,#FCC509A3,#FCC509B8),url('/assets/images/aktuelles/titelbild.jpg') bottom right / cover;">
         <div class="container">
             <div class="hero-inner">
                 <div class="hero-card">
                     <h1>Aktuelles </h1>
                     <h3>
                         Hier finden sie die neuesten Informationen </br>
-                        und bleiben so immer up to date.
+                        und bleiben so immer up-to-date.
```

Comment:
In my opinion the hyphen look weird

Reply:
both are correct but i think this is the more german way